lib/modules/manager/gomod/extract.ts

```diff
@@ -70,10 +70,17 @@ export function extractPackageFile(content: string): PackageFile | null {
         deps.push(dep);
       }
       const requireMatch = regEx(/^require\s+([^\s]+)\s+([^\s]+)/).exec(line);
-      if (requireMatch && !line.endsWith('// indirect')) {
-        logger.trace({ lineNumber }, `require line: "${line}"`);
-        const dep = getDep(lineNumber, requireMatch, 'require');
-        deps.push(dep);
+      if (requireMatch) {
+        if (line.endsWith('// indirect')) {
+          logger.trace({ lineNumber }, `indirect line: "${line}"`);
+          const dep = getDep(lineNumber, requireMatch, 'indirect');
+          dep.enabled = false;
```

Comment:
```suggestion
          dep.enabled = false;
          dep.skipReason = 'disabled';
```

so until sombody modifies `updateDependency` to support `indirect` deps, nobody should be able to enable via package rule

Reply:
Shall we fix that at the same time? I'm assuming @jamietanna wants to update too